### PR TITLE
Issue 2840: Hyperlinks added to hdfs.yml and zookeeper.yml

### PR DESCRIPTION
**Pull request by Aparnarr**
**Change log description**  
Hyperlinks added to hdfs.yml and zookeeper.yml in docker-swarm documentation

**Purpose of the change**  
 Fixes #2840

### CODE REVIEW

File: documentation/src/docs/streamcuts.md [hunk 1]
```diff
@@ -92,4 +92,4 @@ It must be noted that passing ```StreamCut.UNBOUNDED``` to startStreamCut and en
 current head of stream and the current tail of the stream, respectively.
 
 
-We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/{{variable.pravega_samples_verion}}/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
+We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/v0.3.1/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
```
Comment (tkaitchuck) [inline documentation/src/docs/streamcuts.md]: Why this change?

Comment (Aparnarr) [reply documentation/src/docs/streamcuts.md]: otherwise the link throws 404 error

**Comment by arvindkandhare** [comment]
#2851 fixes the DCO issue. @Aparnarr please verify these changes. Push any changes to `origin:issue-2840-yml-links-signofffixed`. Feel free to close this PR.

File: documentation/src/docs/streamcuts.md (hunk 1 above)
Comment (fpj) [reply documentation/src/docs/streamcuts.md]: This doesn't sound like the right way of solving this. I believe @Prabha-Veerubhotla added that variable (`variable.pravega_samples_verion`) there so that we didn't have to update the link upon every new version. Where is that variable supposed to be updated?

Note that there is a typo in the variable name, so it is possible that the link is broken because of the typo.

Comment (Aparnarr) [reply documentation/src/docs/streamcuts.md]: ok will check

Comment (Aparnarr) [reply documentation/src/docs/streamcuts.md]: version passing through variable is not fixed by prabha

**Comment by Aparnarr** [comment]
Closing this PR in favor of #2930
